build.gradle

```diff
@@ -63,6 +63,8 @@ task integrationTest(type: Test) {
   classpath = sourceSets.integrationTest.runtimeClasspath
 }
 
+check.dependsOn integrationTest
```

Comment:
check is a task to "run all tests" -- so in this case unit tests and integration tests